src/patternfly/components/Label/styles.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../patternfly-utilities";
+
+.pf-c-label {
+  // Component
+  --pf-c-label--PaddingTop: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-label--PaddingBottom: var(--pf-global--spacer--xs);
+  --pf-c-label--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-label--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-label--BorderRadius: var(--pf-global--BorderRadius--sm);
+
+  // Color
+  --pf-c-label--BackgroundColor: var(--pf-global--primary-color--100);
+  --pf-c-label--Color: var(--pf-global--Color--light-100);
+
+  // Font
+  --pf-c-label--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-label--FontWeight: var(--pf-global--FontWeight--regular);
+
+  // Modifiers
+  --pf-c-label--hover--BackgroundColor: var(--pf-global--primary-color--200);
+  --pf-c-label--m-compact--FontSize: var(--pf-global--FontSize--xs);
```

Comment:
Shouldn't there also be `padding-top` and `padding-bottom` changes with the compact modifier? per the design, their value should be `--pf-global--spacer--sm`

Reply:
The designs that I did have to be altered since we only want one sized label now, so Im going to change those spacers to be the same, but thanks for catching that.